src/psmoveservice/Device/Enumerator/VirtualControllerEnumerator.cpp

```diff
@@ -0,0 +1,56 @@
+// -- includes -----
+#include "VirtualControllerEnumerator.h"
+#include "ServerUtility.h"
+#include "assert.h"
+#include "hidapi.h"
+#include "string.h"
+
+//-- Statics
+int VirtualControllerEnumerator::virtual_controller_count= 0;
+
+// -- VirtualControllerDeviceEnumerator -----
+VirtualControllerEnumerator::VirtualControllerEnumerator()
+    : DeviceEnumerator(CommonDeviceState::VirtualController)
+{
+	m_deviceType= CommonDeviceState::VirtualController;
+    m_device_index= 0;
+
+    m_current_device_identifier= "VirtualController_0";
```

Comment:
It looks like all virtual controllers are being identified as "VirtualController_0" which might be the problem when trying to use more than one.